widgets/src/image.rs

```diff
@@ -0,0 +1,174 @@
+use {
+    crate::{
+        makepad_draw::*,
+        widget::*,
+        widget_with_image::*
+    }
+};
+
+live_design!{
+    import makepad_draw::shader::std::*;
+    import makepad_widgets::theme::*;
+    
+    Image = {{Image}} {
+        walk:{
+            width: Fit
+            height: Fit
+        }
+        layout: {
+            // This is important to avoid clipping the image, specially when it is rotated.
+            // In any case, it can be override by the app developer.
+            clip_x: false,
+            clip_y: false
+        }
+        draw_bg: {
+            texture image: texture2d
+
+            instance rotation: 0.0
+            instance opacity: 1.0
+            instance scale: 1.0
```

Comment:
Haven't fully gone through the rotation changes, but should we expect to these instances through overriding `draw_bg ` and have it working with the current code?

I tried this:

```rs
  <Image> {
      source: dep("crate://self/resources/tinrs.png"),
      walk: {width: (1000 * 0.175), height: (175 * 0.175), margin: 0}
      layout: {padding: 0}
      draw_bg: {
          scale: 1.0
          // rotation: 2.0
      }
  }
```

`scale` works as expected but if I use any `rotation` the image stops displaying, (tried this with the tinrs.png in the ironfish demo)

Reply:
This issue should be fixed now.